server/routes/staticContent/staticContentController.ts

```diff
@@ -69,6 +69,16 @@ export default class StaticContentController {
       template: 'serviceProviderReferrals/sessions/amend',
       description: 'IC-1087 (session details: amend)',
     },
+    {
+      path: '/service-provider/session/feedback/confirmationNextStepESOR',
```

Comment:
A couple of things:

1. Can we make this consistent with the URL structure that's used for the normal confirmation page (see line 54 above)?
2. Shouldn't it be "EOSR" (end of service report), not"ESOR"? But actually I think it would be better to be explicit about the state.

```suggestion
      path: '/service-provider/sessions/feedback/confirmation-all-sessions-completed',
      template: 'serviceProviderReferrals/sessions/feedback/confirmationAllSessionsCompleted',
      description: 'IC-1177 (session feedback: confirmation - all sessions completed)'
```